src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -0,0 +1,17 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Roman
+  Date: 06.12.2021
+  Time: 13:25
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" %>
+<html>
+<head>
+    <title>Title</title>
+</head>
+<body>
+<h1>Information written to our resource</h1>
+
+</body>
```

Comment:
I don't know if it is important, but... )
````suggestion
<body>
    <h1>Information written to our resource</h1>
</body>
````

Reply:
I agree that it is better to use correct indentation